### ISSUE TITLE
put scope on ref to indicate where the reference applies

### ISSUE THREAD

**Issue by ttasovac** [DARIAH-ERIC/lexicalresources#63]
this would complementary to ref type...
i#m just parking this issue here, laurent and i will elaborate

**Comment by laurentromary** [comment]
e.g. `<ref type='lemma' scope='currentEntry'/>`

**Comment by ttasovac** [comment]
The following is a proposal that's based on Ana's, Rute's and my presentation at the collocation workshop in Sintra + subsequent discussions with Laurent. 

We got rid of `oRef` and we don't want to resurrect it. We have a robust mechanism for cross-references (`xr/ref`) where xr is the container, and the actual reference string is in `ref` + we can add things like `lbl` etc. A while back we added types to `ref` (`entry`, `sense`) to distinguish the type of reference (whether it's pointing to an entry as a whole, or a particular sense). So far, so good.

At the same time, we need a mechanism for collocations (i.e. "transparent", undefined MWEs...), like here:

![Снимок экрана 2019-09-29 в 21.50.02](https://i.imgur.com/aZdsz7h.png)

In `+ da fonte, da torneira`, plus is a ref to the lemma, it is used only in the first collocation, and in the second it is only implied. 

```xml
<sense xml:id="agua.4">
    <!--etc.-->
    <form type="collocations">
        <form type="collocation">
            <orth>
                <xr type="related">
                    <lbl>+</lbl>
                    <ref type="lemma" scope="currentEntry" target="#agua.lemmaid"/>
                    <seg>da fonte</seg>
            </orth>
        </form>
        <pc>,</pc>
        <form type="collocation">
            <orth>
                <xr type="related">
                    <ref type="lemma" scope="currentEntry" target="#agua.lemmaid"/>
                    <seg>da torneira</seg>
                </xr>
            </orth>
        </form>
    </form>
    <pc>;</pc>
    <!--etc.-->
</sense>
```

The thinking behind it is:

- colloctions of this kind definitely need to be encoded as form/orth
- lexicographic shorthand (+, ~, implied refs) need to be resolvable automatically, i.e. we need to encode things in such a way that we can extract full collocations if we wanted
- we already have a mechanism for cross-references, so we should really use it here as well
- we should discuss the possibility of introducing `@scope` to `<ref>` and then agree on a typology (a la: currentEntry, currentSense, externalEntry, externalSense etc...) to highlight where we are pointing to; this clearly needs more discussion
- `seg` is the default inside `orth` for non-ref strings. of course, one could turn everything into refs, if one wanted to encode every word as a link 

Some issues:

1. I'm not sure is what we should call the `xr` type in these cases. Our current typology is: `synonymy`, `antonymy`, `hypernymy`, `hyponymy`, `meronymy` and a catch-all `related`.  Related we defined as 
```xml
<valItem ident="related">
    <desc>The default reference to another lexical entry when no additional information is
        available.</desc>
</valItem>`
````
  which should, if we keep it for this type of oRef-y use, be changed anyway, because we're not pointing to the "another lexical entry". 

2. I would like to have the option of having `@value `on `<ref>` so that one can explicitly put in the value of the "missing" lemma (i.e. "resolve" the plus sign inline) should one want. The definition of `@value` is quite appropriate for this, but it's not allowed on `ref`. 

3. We need to figure out and agree on how to type forms here (form type = "collocations", form type="collocation", which we will also have to consider together with #65.
